QuizApp/App/SceneDelegate.swift

```diff
@@ -16,8 +16,8 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         window = UIWindow(windowScene: scene)
         
         let viewModel = HomeViewModel()
-        let viewController = HomeViewController(viewModel: viewModel)
-
+        //let viewController = HomeViewController(viewModel: viewModel)
+        let viewController = QuizViewController(viewModel: viewModel, subject: Subject(icon: "", subjectTitle: "პროგრამირება", quizDescription: "აღწერა", quizQuestionCount: 5, questions: []))
```

Comment:
Why do you need to pass all this hardcoded items?

Reply:
it's for testing only, will be removed after I add logic